1_space_odyssey/js/therese-9000/app.js

```diff
@@ -112,7 +125,73 @@ console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, ha
        cannot go outside, console log the correct log based on that condition. 
 */
 if (hasJetPack === true) {
-    console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+  console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+} else {
+  console.log(
+    "JetPack NOT enabled, please enable jetPack before doing a space walk"
+  );
+}
+
+/*
+ 3. METEOR SHOWER::::::
+    On your way to Jupiter you encounter a massive meteor shower that could damage the ship, you need to redirect the ship to a new safe location.
+    a) Using a for loop iterate through the responses array and choose the appropriate response to get out of danger.
+      create an if condition in your loop to print only 1 response using a comparison operator.
+*/
+console.log("\u{1F327} METEOR SHOWER::::::");
+
+var responses = [
+  "Call basecamp and ask for assistance, but they are far away",
+  "Do nothing, just push through",
+  "initiate core thrusters, punch 180degrees into dash-nav and warp away from danger",
+  "ask humanoid-9000 to fly out there and shield the ship from the debris, this action could harm the humanoid"
+];
+
+// write your code here, use the console log beneath
+
+for (var i = 0; i <= 3; i++) {
```

Comment:
:-) close. 

Your conditional should `return` the length of the array.
like this:
`for (var i = 0; i <= responses.length; i++) `

this: `responses.length` will return 4 in this example array.

what if your array had 1000 indexes, your code as it is now would only loop 3 times, when it would need to loop 1000 times.